README.md

```diff
@@ -60,6 +60,22 @@ when Ecto gets more complex field selection support we will go further to only q
 
 You will need to handle filtering yourself, the filter is just a map with key=value.
 
+## Configuration
+
+There are several options available through configuration.
+These include switching from underscores to dashes in serialization, customizing the host name, and customizing the URL scheme.
+
+Example:
+
+```elixir
+config :jsonapi,
+  host: "www.someotherhost.com",
+  scheme: "ftp",
```

Comment:
I think `https` would be a better example here.  JSONAPI and `ftp` don't typically work together.